reactor-test/src/main/java/reactor/test/util/TestLogger.java

```diff
@@ -83,16 +85,28 @@ public boolean isTraceEnabled() {
 
 	@Override
 	public synchronized void trace(String msg) {
-		this.log.format("[TRACE] (%s) %s\n", Thread.currentThread().getName(), msg);
+		if(logCurrentThreadName){
```

Comment:
looking at this, I'm thinking maybe a second method like `private String format` could be used in all the log methods, in order to mutualize this if/else choice, wdyt?

the new method would get two strings as input: the prefix (`[TRACE]` and the other variants) and the user-provided message / `format(format, arguments)` result. it would then decide whether or not to put the thread name in the middle, centralizing that decision.

Reply:
Thanks for the suggestion, that way is much cleaner.